homeassistant/components/zha/core/store.py

```diff
@@ -13,7 +13,7 @@
 from homeassistant.helpers.storage import Store
 from homeassistant.loader import bind_hass
 
-from .typing import ZhaDeviceType
+from .device import ZHADevice
```

Comment:
Let's move that to `if TYPE_CHECKING`, too. Just proactively.

Reply:
done